Util/ClassUtils.php

```diff
@@ -22,10 +20,25 @@ private function __construct()
      */
     public static function getClass($object): string
```

Comment:
What about type-casting `object` here?

Reply:
@garak did you mean type-hinting? 
`public static function getClass(object $object): string`
or we should expect something different than original object here?

Reply:
We expect an object, as suggested by variable name and by using `get_class` on it. So, passing a non-object was already not valid, but without type hint could be unnoticed. I think it's better to  be explicit and to fail with an error in such case